onnxruntime/core/providers/mkldnn/mkldnn_execution_provider.h

```diff
@@ -37,6 +39,28 @@ class MKLDNNExecutionProvider : public IExecutionProvider {
   }
 
   virtual std::shared_ptr<KernelRegistry> GetKernelRegistry() const override;
+
+  std::shared_ptr<mkldnn::memory> Get_weight_memory(std::string weightName) {
```

Comment:
please follow coding conventions:
https://github.com/Microsoft/onnxruntime/blob/master/docs/Coding_Conventions_and_Standards.md

function names should be GetWeighstMemory(), SetWeightsMemory(), ReorderWeights() etc.

Reply:
please also fix the style/formatting by using clang-format with google c++ style rules.

Reply:
enabled clang support in vs2017